raw-data/EA/EA_inventories.tsv

```diff
@@ -5019,7 +5019,7 @@ InventoryID	LanguageCode	LanguageName	Segment
 2356	chx	Chantyal	d
 2356	chx	Chantyal	dz
 2356	chx	Chantyal	ɡ
-2356	chx	Chantyal	m
+2356	chx	Chantyal	m 
```

Comment:
There is trailing white space in the raw data, which I didn't remove previously because this gets handled by the aggregation script and the IPA segment correspondences file. I've removed the whitespace for consistency though. Good catch!